central/reports/common/report_formatter.go

```diff
@@ -150,7 +152,12 @@ func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults [
 
 	var zipBuf bytes.Buffer
 	zipWriter := zip.NewWriter(&zipBuf)
-	zipFile, err := zipWriter.Create(fmt.Sprintf("RHACS_Vulnerability_Report_%s.csv", time.Now().Format("02_January_2006")))
+	reportName := fmt.Sprintf("RHACS_Vulnerability_Report_%s.csv", time.Now().Format("02_January_2006"))
+	log.Infof("configName %s", configName)
+	if configName != "" {
+		reportName = fmt.Sprintf("RHACS_Vulnerability_Report_Config_%s_%s.csv", configName, time.Now().Format("02_January_2006"))
```

Comment:
Let's keep the format `RHACS_Vulnerability_Report_<config_name>_<date>.csv` . No need to include the extra `_Config`.

I think we should have a limit on the length of config name. Like 80 characters maybe. If name is longer than that, then we can just add a `...` after selecting 80 chars.